bindings/python/src/OpenSpaceToolkitPhysicsPy/Units.cpp

```diff
@@ -12,6 +12,7 @@
 #include <OpenSpaceToolkitPhysicsPy/Units/ElectricCurrent.cpp>
 #include <OpenSpaceToolkitPhysicsPy/Units/Time.cpp>
 #include <OpenSpaceToolkitPhysicsPy/Units/Length.cpp>
+#include <OpenSpaceToolkitPhysicsPy/Units/Mass.cpp>
```

Comment:
Import is in reverse order:

```suggestion
#include <OpenSpaceToolkitPhysicsPy/Units/Time.cpp>
#include <OpenSpaceToolkitPhysicsPy/Units/Mass.cpp>
#include <OpenSpaceToolkitPhysicsPy/Units/Length.cpp>
```